app/controllers/protocols_controller.rb

```diff
@@ -129,17 +129,21 @@ def linked_children
       records = records.where(parent_id: @protocol.published_versions)
                        .or(records.where(parent_id: @protocol.id))
     end
-    records.preload(my_module: { experiment: :project }).distinct
-
-    render json: records.map { |record|
-      {
-        my_module_name: record.my_module.name,
-        experiment_name: record.my_module.experiment.name,
-        project_name: record.my_module.experiment.project.name,
-        my_module_url: protocols_my_module_path(record.my_module),
-        experiment_url: my_modules_path(experiment_id: record.my_module.experiment.id),
-        project_url: experiments_path(project_id: record.my_module.experiment.project.id)
-      }
+    records = records.preload(my_module: { experiment: :project }).distinct.order(updated_at: :desc).page(params[:page]).per(10)
```

Comment:
This will fetch always 10 records per page. Does not we support that users select how many records per page they want?